DEPENDENCIES

```diff
@@ -232,20 +233,27 @@ Prerequisites for building the OpenDataPlane (ODP) API
    # reboot, before trying to run e.g. 'make distcheck'.
    sudo ldconfig
 
-4.3 Cross compile of Cunit
+4.3 Cross compile of CUnit
 
    $ git svn clone http://svn.code.sf.net/p/cunit/code/trunk cunit-code
    $ cd cunit-code
    $ ./bootstrap
    $ ./configure --host=arm-linux-gnueabihf --prefix=/home/${USER}/src/install-cunit
 
-4.4 Using Cunit with ODP
-   $ Add the configuration option to the regular configuration options
-   # Use the default path ...
-   ./configure  --enable-cunit
+4.4 Using CUnit with ODP
 
-   # ... OR the user defined path.
-   ./configure  --with-cunit-path=/home/<my_cunit_path>
+   Configure will automatically look for CUnit if validation testsuite is
+   enabled. By default it uses pkg-config to locate CUnit. Usually no
+   additional configuration will be required. Few corner cases:
+
+   # Systemwide CUnit if configure errors out with 'missing Version' message
+   ./configure CUNIT_CFLAGS="-I/usr/include" CUNIT_LIBS="-lcunit"
```

Comment:
can configure take care about this step?

Reply:
Not really, that is a problem. Technically it should be a bug opened against distro to fix cunt.pc. I can resurrect `--with-cunit-path` option, if that makes sense from your POV.

Reply:
I prefer better verbose message from ./configure to avoid reply to the same questions :) I.e. or resurrect  --with-cunit-path or move that text to ./configure